### DISCUSSION TITLE
How to `bun test **/test.tsx`

### DISCUSSION THREAD

**Discussion by giovannivocale-molar** [oven-sh/bun#5880] [General]
All my test files are named as such:
```tsx
src/components/ComponentName/test.tsx
```

How can I let bun find test files named `test.tsx`?

**Comment by Almenon** [comment]
Tests need ".test", "\_test\_", ".spec" or "\_spec\_" in the filename (ex: "MyApp.test.ts"). I searched for a option to pick up test files in different formats but didn't find one, so I suggest renaming your files to match the convention.
